### PR TITLE
Fix instructions to view npc logs when >1 pod exists

### LINKED ISSUE #2693: weave-npc troubleshooting command fails on >1 pod
From [Slack](https://weaveworks.slack.com/archives/weave-users/p1481579225000303):

in our case, we have two kube clusters so i was trying to run the command like this:
```
kubectl --context=XXXXXXX logs $(kubectl --context=XXXXXXX get pods --all-namespaces | grep weave-net | awk '{print $2}') --namespace=kube-system weave-npc
```
but i get an error:
```
error: logs POD [-c CONTAINER]
See 'kubectl logs -h' for help and examples.
```

here is the output for that command:
```
kubectl --context=X get pods --all-namespaces | grep weave-net
kube-system   weave-net-5iucz                                         2/2       Running            0          5d
kube-system   weave-net-8uj8u                                         2/2       Running            0          5d
kube-system   weave-net-ijd2p                                         2/2       Running            0          5d
kube-system   weave-net-iwxxh                                         2/2       Running            0          5d
kube-system   weave-net-qwv7u                                         2/2       Running            0          5d
kube-system   weave-net-uirdt                                         2/2       Running            0          5d
kube-system   weave-net-vpe2k                                         2/2       Running            0          5d
kube-system   weave-net-vzk4d                                         2/2       Running            0          5d
kube-system   weave-net-x075p                                         2/2       Running            0          5d
```

### PR DESCRIPTION
**Pull request by bboreham**
Fixes #2693 

cc @abuehrle for docs review

### CODE REVIEW

File: site/kube-addon.md [hunk 1]
```diff
@@ -77,10 +77,21 @@ definition](http://kubernetes.io/docs/api-reference/extensions/v1beta1/definitio
...
+To do this, first you have to find the name of the Weave Net pod running on the relevant host:
+
+```
+$ kubectl get pods -n kube-system -o wide | grep weave-net
+weave-net-08y45                  2/2       Running   0          1m        10.128.0.2   host1
+weave-net-2zuhy                  2/2       Running   0          1m        10.128.0.4   host3
+weave-net-oai50                  2/2       Running   0          1m        10.128.0.3   host2
+```
+
+Pick the one you need - e.g. if you want to look at host2 then pick `weave-net-oai50`, and do:
```
Comment (abuehrle) [inline site/kube-addon.md]: Select the relevant container, for example, if you want to look at host2 then pick `weave-net-oai50` and run:

File: site/kube-addon.md [hunk 2]
```diff
@@ -77,10 +77,21 @@ definition](http://kubernetes.io/docs/api-reference/extensions/v1beta1/definitio
 
 ###<a name="blocked-connections"></a> Troubleshooting Blocked Connections
 
-If you suspect that legitimate traffic is being blocked by the Weave Network Policy Controller, check the `weave-npc` container's logs:
+If you suspect that legitimate traffic is being blocked by the Weave Network Policy Controller, check the `weave-npc` container's logs.
+
```
Comment (abuehrle) [inline site/kube-addon.md]: If you suspect that legitimate traffic is being blocked by the Weave Network Policy Controller, the first thing to do is to check....

**Comment by awh** [comment]
#### Documentation changes:
modified [kube-addon](https://dev-weavewww.pantheonsite.io/docs/net/2693-npc-log-instructions/kube-addon/)

**Comment by awh** [comment]
#### Documentation changes:
modified [kube-addon](https://dev-weavewww.pantheonsite.io/docs/net/2693-npc-log-instructions/kube-addon/)

**Pull request merged**